modules/displayPage.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable*/
```

Comment:
Same here. Please don't disable linters. There is no reason to disable it here. 👍 Just remove it, you are good to go 😎